llvm/test/CodeGen/RISCV/xmips-cbop.ll

```diff
@@ -49,3 +49,47 @@ define void @prefetch_inst_read(ptr noundef %ptr) nounwind  {
   tail call void @llvm.prefetch.p0(ptr nonnull %arrayidx, i32 0, i32 0, i32 0)
   ret void
 }
+
```

Comment:
These tests produce identical code without the changes to RISCVRegisterInfo.cpp.